packages/flutter/test/material/app_test.dart

```diff
@@ -25,6 +25,23 @@ class StateMarkerState extends State<StateMarker> {
   }
 }
 
+class PopWidget extends StatelessWidget {
+  final dynamic value;
+  final String name;
+  final BuildContext context;
+
+  const PopWidget(this.context, this.name, this.value);
```

Comment:
Yeah, I thought it was pretty crazy too, I was just looking for a way to delay the pop until after building was done.  Thanks for the suggestion: it's much simpler now.